eng/targetframeworksuffix.props

```diff
@@ -9,6 +9,7 @@
     <TargetPlatformIdentifier>$(TargetFrameworkSuffix)</TargetPlatformIdentifier>
     <TargetPlatformVersion>1.0</TargetPlatformVersion>
     <TargetPlatformMoniker>$(TargetFrameworkSuffix),Version=$(TargetPlatformVersion)</TargetPlatformMoniker>
+    <SupportedOSPlatformVersion>0.0</SupportedOSPlatformVersion>
```

Comment:
I recommend leaving a comment here indicating what this does; not many folks are familiar with this property (yet).